test/hack/checkLabels/main.go

```diff
@@ -33,25 +41,34 @@ func parseInhibitionFile(fileName string) (alertConfig.Config, error) {
 	return inhibitions, nil
 }
 
-// Return the list of target and source labels from the alertmanager config file
-func getTargets(config alertConfig.Config) ([]string, []string) {
-	var targetMatchers []string
-	var sourceMatchers []string
+// Return either the list of target or a specific source label from the alertmanager config file
+func getTargetsAndSources(config alertConfig.Config, target string) ([]string, string) {
```

Comment:
Originally, I had 2 distinct functions but to avoid code duplication (especially with the 2 for loops) I merged them into a single one and had to use a dirty trick to make it work. 
Would you still prefer to have 2 functions even if it means code duplication ?

Reply:
As discussed, function stayed as it is but I added comments to better explain its purpose.